### DISCUSSION TITLE
User-Permissions dependency??

### DISCUSSION THREAD

**Discussion by Jose10go** [VirtusLab-Open-Source/strapi-plugin-comments#149] [Ideas]
Hi! I a would like to use the plugin in a strapi instance without the users-permision plugin... Is there a reason for that dependency?
In the server boostrap function (around line 20) you can read :

```javascript
// Check if the plugin users-permissions is installed because the navigation needs it
  if (Object.keys(strapi.plugins).indexOf("users-permissions") === -1) {
    throw new Error(
      "In order to make the comments plugin work the users-permissions plugin is required"
    );
  }
  ```
  What means that the navigation needs the permission plugin? It would be useful to have a way to opt-out the users plugin.

**Comment by cyp3rius** [comment]
Yep, we're using RBAC and it's a must if you would like to use the plugin.
